cli/pcluster/pcluster.py

```diff
@@ -577,7 +577,36 @@ def instances(args):
         LOGGER.info("Run 'awsbhosts --cluster %s' to list the compute instances", args.cluster_name)
 
 
-def command(args, extra_args):
+def get_master_server_ip(stack_name, config):
```

Comment:
I'd add the underscore prefix since it is an internal function and the docstrings too.

Reply:
👍🏻 done